dotnet/samples/04.ai.d.chainedActions.listBot/Model/ListState.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.TeamsAI.State;
+
+namespace ListBot.Model
+{
+    public class ListState : TurnState<ConversationState, StateBase, ListTempState> { }
+
+    public class ConversationState : StateBase
+    {
+        private const string _greetedKey = "greetedKey";
+        private const string _listNamesKey = "listNamesKey";
+        private const string _listsKey = "listsKey";
+
+        public bool Greeted
+        {
+            get => Get<bool>(_greetedKey);
+            set => Set(_greetedKey, value);
+        }
+
+        public IList<string>? ListNames
+        {
+            get => Get<IList<string>>(_listNamesKey);
+            set => Set(_listNamesKey, value);
+        }
+
+        public Dictionary<string, IList<string>>? Lists
+        {
+            get => Get<Dictionary<string, IList<string>>>(_listsKey);
+            set => Set(_listsKey, value);
+        }
+    }
+
+    public class ListTempState : TempState
+    {
+        private const string _listsKey = "listsKey";
+
+        public Dictionary<string, IList<string>>? Lists
```

Comment:
Curious where's this temp state used...
I had the same question on devops bot and finally removed this...

Reply:
JS sample use temp state variable in prompt. Can be removed because we use function instead.

Reply:
Removed.